README.md

```diff
@@ -23,7 +23,7 @@ _Note: Heroku preview does not include email or persistent storage._
 ## Install Mattermost
 
 - [Quick Install Guide](https://docs.mattermost.com/getting-started/light-install.html) - Deploy in minutes via Mattermost Omnibus on Ubuntu
-- [Run Mattermost via Docker](https://docs.mattermost.com/install/setting-up-local-machine-using-docker.html) - Launch a Mattermost server instantly to test functionality and build integrations
+- [Docker Deployment](https://docs.mattermost.com/install/install-docker) - Preview Mattermost instantly or deploy via Docker for production use.
```

Comment:
@azigler This docs link is likely to change via https://github.com/mattermost/docs/pull/5539

Reply:
Same here, submitted a preemptive edit!